localstack/services/kms/provider.py

```diff
@@ -466,50 +677,169 @@ def import_key_material(
                 f"Unsupported padding, requested wrapping algorithm:'{import_state.wrapping_algo}'"
             )
 
-        key_material = import_state.key_obj.decrypt(encrypted_key_material, decrypt_padding)
-        key_obj.key_material = key_material
+        key_material = import_state.key.crypto_key.key.decrypt(
+            encrypted_key_material, decrypt_padding
+        )
+        key_to_import_material_to.crypto_key.key_material = key_material
+        LOG.info("Key material: %s", key_material)
```

Comment:
nit: we could probably make that a `debug` log:
```suggestion
        LOG.debug("Imported key material for key %s: %s", key_id, key_material)
```

Reply:
I just removed that logging. Was probably me debugging the code in the process. Logging this information while not logging a lot of other through the rest of operations doesn't make much sense.